docs/Release-Process.md

```diff
@@ -5,44 +5,66 @@ Containers release
 
 ## Release phases
 
-The confidential-containers
-[release process](https://github.com/confidential-containers/community/blob/main/.github/ISSUE_TEMPLATE/release-check-list.md)
-lists the tasks involved in doing a release and these largely break down to three phases:
-- Release candidate selection and testing
-- Cutting releases
-- Post release
-
-### Release candidate testing
-
-In the release candidate selection and testing phase, we ensure that the dependencies we have within the
-confidential-containers projects are updated and that Kata Containers is updated to use these new versions, the
-[Kata Containers Runtime Payload CI](https://github.com/kata-containers/kata-containers/actions/workflows/cc-payload-after-push.yaml)
-image is updated, the operator is updated and the tests pass with all of these.
-
-At this point, we should update the cloud-api-adaptor versions to use these release candidate versions of:
-- The kata-containers source branch that we use in the [podvm `Dockerfiles`](../src/cloud-api-adaptor/podvm/) and the
-[podvm workflows](../.github/workflows), by updating the `git.kata-containers.reference` value in [versions.yaml](../src/cloud-api-adaptor/versions.yaml) to
+In the new Confidential Containers
+[release process](https://github.com/confidential-containers/community/blob/main/.github/ISSUE_TEMPLATE/release-check-list.md),
+the plan is to do a succession of component releases, rather than releasing
+all components at the same time. This means that the peer pods release process needs to slot into the correct place.
+The flow of releases should roughly be:
+1. The [guest-components release](https://github.com/confidential-containers/guest-components/releases) (or a pinned
+version is picked) and [trustee release](https://github.com/confidential-containers/trustee/releases) and [kata-containers](https://github.com/kata-containers/kata-containers) bumps these versions in
+    [versions.yaml](https://github.com/kata-containers/kata-containers/blob/main/versions.yaml) under
+    `externals.coco-guest-components.version`, `externals.coco-trustee` and the `image-rs` crate in the agent's
+    [`Cargo.toml`](https://github.com/kata-containers/kata-containers/blob/main/src/agent/Cargo.toml).
```

Comment:
I have a hard time to understand this (the problem could be me not being a native speaker). "[A] ... release ... bumps $something". I have some rough idea what this means, but maybe thsi could be rephrased in simpler terms?

Reply:
Sure, I'll have a go at making it clearer, and your English is fantastic, so it's a me problem, not you!

Reply:
I hope this helps, otherwise please send it back for more clarification!